### PR TITLE
Allow IPv6 address to be parsed as controller ip address.

### PR DESCRIPTION
**Pull request by kenchiang**
Reviewer: @poolakiran @rlane

### CODE REVIEW

File: modules/OFConnectionManager/module/src/ofconnectionmanager_config.c [hunk 1]
```diff
@@ -223,12 +231,27 @@ parse_controller(struct controller *controller, cJSON *root)
         return err;
     }
 
-    /* TODO validate IP */
+    /* if the ip address contains a colon, it is considered ipv6 */
+    is_ipv6 = false;
+    for (idx = 0; idx < strlen(ip); idx++) {
```
Comment (rlane) [inline modules/OFConnectionManager/module/src/ofconnectionmanager_config.c]: I think you could use `strchr` here.

**Comment by rlane** [comment]
One comment, otherwise looks good.

**Comment by kenchiang** [comment]
thanks for taking a look.  pushed a new commit.

**Comment by rlane** [comment]
LGTM

**Comment by poolakiran** [comment]
LGTM

ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vbs22/abat/2015.01.27.2310-m.indigo.master/abat.log

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vbs22/abat/2015.01.27.2310-m.indigo.master/abat.log
/cc @rlane

**Pull request merged**